scipy/stats/tests/test_multivariate.py

```diff
@@ -497,6 +684,44 @@ def test_lnB(self):
 
         assert_almost_equal(np.exp(_lnB(alpha)), desired)
 
+    def test_random_variates_remain_identical(self):
+        # Ensure random variates are identical to numpy.
+        from numpy.random import RandomState
+        seed = 0
+        cov = np.array([[4, 1], [1, 3]])
+        mu = np.array([1, 2])
+        size_expected = [
+            (1, np.squeeze(RandomState(seed).multivariate_normal(mu, cov, 1))),
+            ((2, 3), RandomState(seed).multivariate_normal(mu, cov, (2, 3)))
+        ]
+        for size, expected in size_expected:
+            assert_allclose(
+                expected,
+                multivariate_normal.rvs(
+                    mean=mu, cov=cov, size=size,
+                    random_state=RandomState(seed)),
+            )
+
+    def test_mean_none_equals_mean_zero_with_inverse_cov(self):
+        # Ensure when mean is set to a zero vector when the corresponding
+        # argument is None and inverse covariance is specifed.
+        dim = 5
+        mean = np.zeros(5)
+        cov = np.diag(np.abs(np.random.randn(dim)))
+        inv_cov = np.diag(1 / np.diag(cov))
+        x = np.random.randn(dim)
+        self._assert_outputs_close(
+            x,
+            multivariate_normal(mean=mean, cov=cov),
+            multivariate_normal(None, cov=None, inverse_cov=inv_cov),
+            {}
+        )
+        self._assert_outputs_close(
+            x,
+            multivariate_normal(mean=mean, cov=cov),
+            multivariate_normal, dict(mean=None, cov=None, inverse_cov=inv_cov)
+        )
```

Comment:
This seems like a good thing to test. I don't remember whether I have something for that.